mantidproject/mantid
# Refactored the creation of FractionalRebinning Quad

Fixes #13411 
Since finding that the quadrilateral needed for Fractional Rebinning is not consistent for each type of conversion, it has been decided that this code should be refactored into the CalculateReflectometry header for each specific conversion.

Each conversion will have its own 'createQuad' method that will allow us to calculated the vertices correctly depending on the type of conversion that has been selected.

Currently CalculateReflectometryQxQz is working as it should and documentation on the help
page has been updated to include images of patch plots for K and P Transformations.

Before testing please ask me for any files that are needed but you cannot find.

To Test:
- Load `data_th_lam.nxs` file into Mantid (find file in comments below)
- Run the scripts featured below individually 
  (making sure to delete all workspaces barre the original data file)
- Compare the Colour fill plots of the Output workspace or the patch plots
   for the dump-vertexes that are produced with the plots in the pdf (found in comments)
- They should be similar plots with the incident beamline having the same orientation.

Scripts for running ConvertToReflectometryQ and Patch plots can be found in the comments below.
